chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/listing/reviewer/edition2015/MeasureComparisonReviewer.java

```diff
@@ -0,0 +1,63 @@
+package gov.healthit.chpl.validation.listing.reviewer.edition2015;
+
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import gov.healthit.chpl.domain.CertifiedProductSearchDetails;
+import gov.healthit.chpl.domain.ListingMeasure;
+import gov.healthit.chpl.permissions.ResourcePermissions;
+import gov.healthit.chpl.util.ErrorMessageUtil;
+import gov.healthit.chpl.validation.listing.reviewer.ComparisonReviewer;
+
+@Component("measureComparisonReviewer")
+public class MeasureComparisonReviewer implements ComparisonReviewer {
+    private ResourcePermissions resourcePermissions;
+    private ErrorMessageUtil msgUtil;
+
+    @Autowired
+    public MeasureComparisonReviewer(ResourcePermissions resourcePermissions,
+            ErrorMessageUtil msgUtil) {
+        this.resourcePermissions = resourcePermissions;
+        this.msgUtil = msgUtil;
+    }
+
+    @Override
+    public void review(CertifiedProductSearchDetails existingListing, CertifiedProductSearchDetails updatedListing) {
+        // checking for the addition of a removed measure.
+        // this is only disallowed if the user is not ADMIN/ONC, so first check the permissions
```

Comment:
I'm assuming this restriction follows the restriction that was in place for the original measure things. Is that right?

Reply:
Yes